titration/ui_state/user_value/buffer_ph.py

```diff
@@ -13,7 +13,7 @@ def save_value(self, value):
         """
         The function to save the titrator's buffer pH
         """
-        self.titrator.buffer_ph = value
+        self.titrator.buffer_ph = float(value)
```

Comment:
Should this be done in the UserValue class? Or do you want to ensure its a float by having it here?

Reply:
I think this should be done in the UserValue class. I'll fix this shortly.

Reply:
This is fixed, ready to be merged